GUI/python source/UMSATSGUI_support.py

```diff
@@ -83,7 +80,12 @@ def init(top, gui, *args, **kwargs):
     logFile1 = None
     logFile2 = None
 
-    top_level.bind_all('<<newSerialData>>', updateDisplay)
+    date = str(datetime.datetime.today().month) + "-" + str(datetime.datetime.today().day) + "_" + str(datetime.datetime.today().hour) + "h" + str(datetime.datetime.today().minute) + "m"
```

Comment:
I think you can use strftime to format your date string, instead of going to this trouble: https://stackoverflow.com/questions/311627/how-to-print-a-date-in-a-regular-format